app/controllers/api_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApiController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  private
+  def authenticated?
+    authenticate_or_request_with_http_basic {|username, password_digest| User.where( username: username, password_digest: password_digest).present? }
```

Comment:
This code is fine.. but multi line blocks should use `do`...`end` instead of curly brackets to make this more readable.  See [ruby style guide](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)